src/Aspire.Dashboard/Configuration/ValidateDashboardOptions.cs

```diff
@@ -55,6 +55,11 @@ public ValidateOptionsResult Validate(string? name, DashboardOptions options)
                 break;
         }
 
+        if (options.Frontend.ConsoleLogHistoryLimit <= 0)
```

Comment:
Should zero be allowed? Maybe someone wants to disable showing console logs.

I'm not sure whether OTEL telemetry limits can be zero. I don't think I ever tested.